packages/core/src/Dialog/Dialog.spec.tsx

```diff
@@ -0,0 +1,71 @@
+import { render, screen } from '@testing-library/react'
+import React from 'react'
+import { Dialog } from '..'
+
+const triggerText = 'Open Dialog'
+const contentText = 'Dialog Content'
+
+describe('Dialog', () => {
+  it('renders the trigger node', () => {
+    render(
+      <Dialog triggerNode={<button>{triggerText}</button>} ariaDescription='Description' ariaTitle='Title'>
+        {contentText}
+      </Dialog>
+    )
+
+    const triggerNode = screen.getByText(triggerText)
+    expect(triggerNode).toBeInTheDocument()
+  })
+
+  it('renders the dialog content when trigger is clicked', () => {
+    render(
+      <Dialog triggerNode={<button>{triggerText}</button>} ariaDescription='Description' ariaTitle='Title'>
+        {contentText}
+      </Dialog>
+    )
+
+    const triggerNode = screen.getByText(triggerText)
+    triggerNode.click()
+
+    const contentNode = screen.getByText(contentText)
+    expect(contentNode).toBeInTheDocument()
+  })
```

Comment:
These are fine to leave around for now, but I'll probably end up deleting them in favor of some Chromatic tests in my next PR 😅